pysam/libcalignedsegment.pyx

```diff
@@ -2448,8 +2442,9 @@ cdef class PileupColumn:
             # warning: there could be problems if self.n and self.buf are
             # out of sync.
             for x from 0 <= x < self.n_pu:
-                pileups.append(makePileupRead(&(self.plp[0][x]),
-                                              self._alignment_file))
+                pileups.append(
```

Comment:
I should probably disclose that my desktop monitor is 34" wide and can show 3 screens with >140 columns side-by-side.  I realize that is a bit extreme, but wonderful for running IGV.